### DISCUSSION TITLE
Announcement Function

### DISCUSSION THREAD

**Discussion by Licidil** [sct/overseerr#3483] [Feature Requests]
Is it possible to add an Announcement Centre function, where the admin(s) can set an announcement for users to see when the login/open Overseerr? Kinda like if you have to make an announcement that a service, like sonarr or radarr, is offline, or if a root folder is unavailable or getting too full

**Comment by amddeus** [comment]
That be cool

**Comment by prospero21** [comment]
I would very much like this feature aswell. Overseerr is one of the most used selfhosted services my users utilize, so it would be a great way to be able to interact with them.
